### ISSUE TITLE
Create Quasar build config

### ISSUE THREAD

**Issue by WalterMoar** [bcgov/ppr-deprecated#4]
Before we can put anything on the front end, we need a build config that works with Quasar.

**Comment by WalterMoar** [comment]
I'm de-scoping the pipeline work from this ticket, just to make it a bit more bite-sized.

**Comment by WalterMoar** [comment]
Built manually and deployed to dev from a personal branch (quasar). Will keep this ticket open until some filesystem layout and build decisions are made and the branch is merged.

**Comment by WalterMoar** [comment]
Putting this on hold until decisions are made about the front end.

**Comment by WalterMoar** [comment]
Closing this - we're going to use CLI 4.
